recipes/UniFunc/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "unifunc" %}
+{% set version = "1.3.4" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/UniFunc/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 85e509127cdfcb9e1cda45eb0e283548ee9766395dea44df625bef91c5e0c739
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
```

Need to add an [entry_point](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds):
```suggestion
  noarch: python
  entry_points:
    - unifunc=unifunc.__main__:main

```